### PR TITLE
Update telemetry

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,3 +134,8 @@ Issue tracker (readonly): https://jira.sonarsource.com/browse/SLVSCODE
 Copyright 2017-2020 SonarSource.
 
 Licensed under the [GNU Lesser General Public License, Version 3.0](http://www.gnu.org/licenses/lgpl.txt)
+
+## Data and telemetry
+
+This extension collects usage data and sends it to SonarSource to help improve our products and services. Collection of telemetry is controlled via the setting: "sonarlint.disableTelemetry". Click [here](telemetry-sample.md) to see a sample of the data that are collected.
```
Comment (jblievremont) [inline README.md:140]:
```suggestion
This extension collects usage data and sends it to SonarSource to help improve our products and services. Collection of telemetry is controlled via the setting: `sonarlint.disableTelemetry`. Click [here](telemetry-sample.md) to see a sample of the data that are collected.
```

**Comment by jblievremont** [comment]
Hey @christophelevis @n-b-sonar, any other comment on this PR? We'd like to be able to send all this to dogfood early next week and release e.g mid-week. Thanks! :pray:

**Comment by n-b-sonar** [comment]
This is a bit too out of context for me to provide any constructive input (am missing the Why/What).

**Comment by jblievremont** [comment]
* The Why and What of the change are described in [SLVSCODE-124](https://jira.sonarsource.com/browse/SLVSCODE-124): we want to understand what JRE users are running to make sure we (and they) are ready when analyzers start moving to Java 11
* Why @christophelevis added you as a reviewer: I believe it is because the responsibility for the wording of the README, which also serves as the marketplace description, is shared between the Marketing and SonarLint teams

Does this help?

File: README.md [hunk 2]
```diff
@@ -135,3 +135,8 @@ Issue tracker (readonly): https://jira.sonarsource.com/browse/SLVSCODE
...
```
Comment (n-b-sonar) [inline README.md:141]:
```suggestion
This extension collects anonymous usage data and sends it to SonarSource to help improve SonarLint functionality.  No source code nor IP address is collected, and SonarSource does not share the data with anyone else. Collection of telemetry is controlled via the setting: "sonarlint.disableTelemetry". Click [here](telemetry-sample.md) to see a sample of the data that are collected.
```

**Pull request merged**